elements/lisk-api-client/test/unit/account.spec.ts

```diff
@@ -65,7 +65,7 @@ describe('account', () => {
 			});
 		});
 
-		describe('get', () => {
+		describe('get from address as buffer', () => {
```

Comment:
probably we can structure it like
```
describe('get', () => {
   describe('account by address as string', () => {
      it('should invoke app:getAccount', async () => {});
   });
   
   describe('account by address as buffer', () => {
      it('should invoke app:getAccount', async () => {});
   });
});
```
?
if yes, then same comment for below tests as well.